### PR TITLE
Update change-site-address.md

### PR DESCRIPTION
**Pull request by PramodBalusu**

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointOnline/change-site-address.md [hunk 1]
```diff
@@ -144,6 +144,9 @@ Flows will need to be recreated after the site address change.
 You need to reconnect the app or apps to your data source. 
 Start by deleting the existing SharePoint connections to any lists you may have before you reconnect to your data. Once you've done that, reconnect your app to the SharePoint lists you were using. Most fields should update automatically. Certain types seem to have trouble updating and can be fixed by simply deleting the field and then undoing the delete.
 
+**Custom Forms Created in PowerApps**<br>
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/change-site-address.md:147]:
```suggestion
**Custom forms created in PowerApps**<br>
```

File: SharePoint/SharePointOnline/change-site-address.md [hunk 2]
```diff
@@ -144,6 +144,9 @@ Flows will need to be recreated after the site address change.
...
+You need to recreate the Custom Form after the site address change.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/change-site-address.md:148]:
```suggestion
You need to recreate the custom form after the site address change. 
```

**Comment by scanum** [comment]
@kaarins PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**